content/docs/cml/self-hosted-runners.md

```diff
@@ -83,24 +83,27 @@ you do it your way :)
-| `--cloud-hdd-size`    | HDD size in GB.                                                                                               | Accepts integer values                                                                                         |
-| `--cloud-ssh-private` | Your private RSA SSH key. If not provided will be generated by the Terraform-provider-Iterative.              | Accepts string                                                                                                 |
-| `--cloud-spot`        | Request a spot instance                                                                                       | Boolean                                                                                                        |
-| `--cloud-spot-price`  | Spot max price. If not specified it takes current spot bidding pricing.                                       | default: `-1`                                                                                                  |
-| `-h`                  | Show help                                                                                                     | Boolean                                                                                                        |
+| Argument                 | Description                                                                                                   | Values                                                                                                         |
+| ------------------------ | ------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------- |
+| `--version`              | Show version number                                                                                           | Boolean                                                                                                        |
+| `--labels`               | Comma delimited runner labels                                                                                 | Default: `cml`                                                                                                 |
+| `--idle-timeout `        | Time in seconds for the runner to be waiting for jobs before shutting down. 0 waits forever.                  | Default: `300`                                                                                                 |
+| `--name`                 | Name displayed in the repo once registered                                                                    | Default: `cml-` followed by a unique identifier, i.e. `cml-cfwj9rrari`                                         |
```

Comment:
I noticed the `Values` column is a bit inconsistent. Sometimes it has a var type e.g. `String` and other times (like here) it specifies the default string instead. Further down there are some defaults in the `Description` too. Would be good to make it consistent one way or another.

Commenting post-merge since it's unrelated to the original intention of the PR.

Reply:
I'm inclined to think that the best long-term solution would be replacing this section with automatically generated documentation as per my comment on https://github.com/iterative/cml/issues/361.

In the meanwhile, I'll take a look to the table tomorrow and try to unify the current row style sample book.

Reply:
Resolved through #2321